suite/tests/CMakeLists.txt

```diff
@@ -2647,6 +2647,10 @@ if (CLIENT_INTERFACE)
       set(tool.drcacheoff.opcode_mix_depends tool.drcacheoff.simple)
       set(tool.drcacheoff.opcode_mix_depends tool.drcacheoff.filter)
 
+      torunonly_drcacheoff(view ${ci_shared_app} ""
+        "@-simulator_type@view" "")
+      set(tool.drcacheoff.view_depends tool.drcacheoff.opcode_mix)
```

Comment:
Please include a comment explaining why it depends.

Reply:
Done.